### PR TITLE
Added search bar

### PR DESCRIPTION
**Pull request by supertata**
Added search bar to search questions' title.

### CODE REVIEW

**Review by elsahib — COMMENTED** [review]
Nice work

File: client/src/Components/List.js [hunk 1]
```diff
@@ -3,11 +3,32 @@
 import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
 import ShowContext from "./ShowContext";
+import Search from "./Search";
 
 const List = ({ data, postComment, getComments }) => {
+  const [searchTerm, setSearchTerm] = useState("");
+  console.log(searchTerm);
```
Comment (anthonytranDev) [inline client/src/Components/List.js]: **Style**

If this goes into the dev, would be nice to keep things clean and remove this

File: client/src/Components/List.js [hunk 2]
```diff
@@ -3,11 +3,32 @@
...
+  const searchChange = (searchValue) => {
+    setSearchTerm(searchValue);
+  };
+
+  const filteredQuestions = data.filter((item) =>
```
Comment (anthonytranDev) [inline client/src/Components/List.js]: **Style**

Can written be written, just a suggestion though. Nice you kept this very clean already
```js
const filteredQuestions = data.filter(({ title }) => titles.includes(searchTerm))
```

File: client/src/Components/List.js [hunk 3]
```diff
@@ -3,11 +3,32 @@
...
+      {!searchTerm &&
+        data.map((item) => (
+          <ShowContext
+            postComment={postComment}
+            getComments={getComments}
+            key={item.id}
+            {...item}
+          />
+        ))}
+      {filteredQuestions.map((item) => (
```
Comment (anthonytranDev) [inline client/src/Components/List.js]: **Refactor**

Not sure if it's the naming of `data` and `filteredQuestion`, but either way, I think we can make this into a dumb component possibly. Lines 23 - 38

File: client/src/Components/Search.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            placeholder="Search"
+            className="mr-sm-2"
+            onChange={handleChange}
+          />
+          <Button variant="outline-success">Search</Button>
+        </Form>
+      </Navbar.Collapse>
+    </Navbar>
+  );
+};
```
Comment (anthonytranDev) [inline client/src/Components/Search.js]: **Semantics**

Should the navigation and search field be in the same component or can we split this into 2 or more components

**Review by anthonytranDev — COMMENTED** [review]
**Demo**

Like I said, would be awesome if you guys can demo this (as I can't currently run in on my machine). Gives me more context and I can give you more context as well about the comments I've made

**Review by halitos — APPROVED** [review]

**Pull request merged**